spec/helpers/middleman/menu_spec.rb

```diff
@@ -42,18 +42,11 @@
       end
 
       # These are just hardcoded values.
-      expect(rows_length).to eq(5)
+      expect(rows_length).to eq(4)
       expect(columns_length).to eq(8)
       expect(links_length).to eq(24)
     end
 
-    it "should have a partial for blogs in place of a column" do
-      menus = @app.get_menu_config()
-      resources = menus[3]
-
-      expect(resources[:rows][1]).to eq({ visible_mobile: false, partial: 'partials/header/blog' })
-    end
```

Comment:
So we're removing the "From the Blog" bit, is that right?  LGTM…I assume people are happy with that (and I don't think we could ship it any other way), just want to ask.

Reply:
yup - brought it up in Standup yesterday and Adriana is going to/has let the Marketing peeps know - yeah that's what I thought too!